statsmodels/tsa/stattools.py

```diff
@@ -1295,6 +1295,149 @@ def levinson_durbin_pacf(pacf, nlags=None):
+
+    Parameters
+    ----------
+    resid : array_like
+        Residuals of a time series model.
+    subset_length : {int, float}
+        Length of the subsets to test (h in Notes below).
+        If a float in 0 < subset_length < 1, it is interpreted as fraction.
+        Default is 1/3.
+    alternative : str, 'increasing', 'decreasing' or 'two-sided'
+        This specifies the alternative for the p-value calculation. Default
+        is two-sided.
+    use_f : bool, optional
+        Whether or not to compare against the asymptotic distribution
+        (chi-squared) or the approximate small-sample distribution (F).
+        Default is True (i.e. default is to compare against an F
+        distribution).
+
+    Returns
+    -------
+    output : ndarray
+        An array with `(test_statistic, pvalue)` for each endogenous
+        variable. The array is then sized `(k_endog, 2)`. If the function is
+        called as `het = breakvar_heteroskedasticity_test(resid)`,
+        then `het[0]` is an array of size 2 corresponding to the first
+        endogenous variable, where `het[0][0]` is the test statistic,
+        and `het[0][1]` is the p-value.
+
+    Notes
+    -----
+    The null hypothesis is of no heteroskedasticity. That means different
+    things depending on which alternative is selected:
+
+    - Increasing: Null hypothesis is that the variance is not increasing
+        throughout the sample; that the sum-of-squares in the later
+        subsample is *not* greater than the sum-of-squares in the earlier
+        subsample.
+    - Decreasing: Null hypothesis is that the variance is not decreasing
+        throughout the sample; that the sum-of-squares in the earlier
+        subsample is *not* greater than the sum-of-squares in the later
+        subsample.
+    - Two-sided: Null hypothesis is that the variance is not changing
+        throughout the sample. Both that the sum-of-squares in the earlier
+        subsample is not greater than the sum-of-squares in the later
+        subsample *and* that the sum-of-squares in the later subsample is
+        not greater than the sum-of-squares in the earlier subsample.
+
+    For :math:`h = [T/3]`, the test statistic is:
+
+    .. math::
+
+        H(h) = \sum_{t=T-h+1}^T  \tilde v_t^2
+        \Bigg / \sum_{t=1}^{h} \tilde v_t^2
+
+    This statistic can be tested against an :math:`F(h,h)` distribution.
+    Alternatively, :math:`h H(h)` is asymptotically distributed according
+    to :math:`\chi_h^2`; this second test can be applied by passing
+    `use_f=False` as an argument.
+
+    See section 5.4 of [1]_ for the above formula and discussion, as well
+    as additional details.
+
+    References
+    ----------
+    .. [1] Harvey, Andrew C. 1990. *Forecasting, Structural Time Series*
+            *Models and the Kalman Filter.* Cambridge University Press.
+    """
+    squared_resid = np.atleast_2d(resid) ** 2
+    nvars, nobs = squared_resid.shape
```

Comment:
Since this is now a general test outside of `statespace`, I will suggest that this should be `(nobs, nvars)` rather than `(nvars, nobs)`. Then you can pass `